### PR TITLE
[ADD] Select advanced stories

### PR DESCRIPTION
**Pull request by MaximeGenevier**
Ajout d'un story pour montrer un exemple de select advanced

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1801/storybook

**Review by shajz — APPROVED** [review]

**Review by jeremie-lucca — CHANGES_REQUESTED** [review]
There are two notable issues: 
- Search feature doesn't work
- If we have Select all / Unselect all buttons, it should be a multiple select with checkbox inside options

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1801/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1801/storybook

File: stories/documentation/forms/select/select-advanced.stories.ts [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+	<label class="textfield">
+		<lu-select
+			class="textfield-input"
+			placeholder="Select advanced"
+			[(ngModel)]="model"
+			multiple
+		>
+			<ng-container *luDisplayer="let value">
+				{{ value.name }}
+			</ng-container>
```
Comment (shajz) [inline stories/documentation/forms/select/select-advanced.stories.ts:21]: missing luDisplayerMultiple (check the code we took inspiration from)

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1801/storybook

File: stories/documentation/forms/select/select-advanced.stories.ts [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+	selector: 'select-advanced-stories',
+	template: `
+	<label class="textfield">
+		<lu-select
+			class="textfield-input"
+			placeholder="Select advanced"
+			[(ngModel)]="model"
+			multiple
+		>
+			<ng-template luDisplayer let-colors luDisplayerMultiple="true">
```
Comment (shajz) [inline stories/documentation/forms/select/select-advanced.stories.ts:19]:
```suggestion
			<ng-template luDisplayer let-colors [luDisplayerMultiple]="true">
```

File: stories/documentation/forms/select/select-advanced.stories.ts [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+	selector: 'select-advanced-story',
+	template: \`
+	<label class="textfield">
+		<lu-select
+			class="textfield-input"
+			placeholder="Select advanced"
+			[(ngModel)]="model"
+			multiple
+		>
+			<ng-template luDisplayer let-colors luDisplayerMultiple="true">
```
Comment (shajz) [inline stories/documentation/forms/select/select-advanced.stories.ts:107]: dito

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1801/storybook

**Review by shajz — DISMISSED** [review]
@jeremie-lucca faut que tu approuves aussi

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1801/storybook

**Pull request merged**